CHANGELOG.rst

```diff
@@ -17,6 +17,18 @@ Unreleased
 * Handle the case where certain toggle names come in as ``None`` when generating summary reports.
 * Add ADR for updating annotations for toggle life expectancy and use cases.
 
+[4.3.0] - 2022-01-31
+--------------------
+
+Removed
+~~~~~~~
```

Comment:
[inform] Although this is a reasonable format for the CHANGELOG, it doesn't match and breaks the formatting. Either this entry needs to be changed, or the other entries need to be made to match.